packages/plans-grid-next/src/components/shared/storage/components/storage-dropdown.tsx

```diff
@@ -106,23 +106,41 @@ const StorageDropdown = ( {
 					siteId,
 				} );
 		}
-	}, [] );
+	}, [
+		defaultStorageOption,
+		planSlug,
+		selectedStorageOptionForPlan,
+		setSelectedStorageOptionForPlan,
+		siteId,
+	] );
 
-	const selectControlOptions = availableStorageAddOns?.map( ( addOn ) => {
-		const addOnStorage = addOn.quantity ?? 0;
-
-		return {
-			key: addOn.addOnSlug,
+	const defaultStorageItem = useMemo(
+		() => ( {
+			key: defaultStorageOption || '',
 			name: (
-				<StorageDropdownOption
-					price={ addOn?.prices?.formattedMonthlyPrice }
-					totalStorage={ planStorage + addOnStorage }
-				/>
+				<StorageDropdownOption price="" totalStorage={ planStorage } />
 			 ) as unknown as string,
-		};
-	} );
+		} ),
+		[ defaultStorageOption, planStorage ]
+	);
+
+	const selectControlOptions = [ defaultStorageItem ].concat(
+		availableStorageAddOns?.map( ( addOn ) => {
```

Comment:
Nit: Feel free to ignore
```suggestion
	const selectControlOptions = [ defaultStorageItem, ...availableStorageAddOns ].map( ( addOn ) => {
```